### PR TITLE
Fix/network switcher

### PR DESCRIPTION
**Pull request by BrandonOCasey**
## Description
* move network switcher code around
* fix broken aspects of the network switcher

## Requirements Checklist
- [x] Feature implemented / Bug fixed
- [ ] Reviewed by Two Core Contributors

### CODE REVIEW

File: utils/switcher/index.html [hunk 1]
```diff
@@ -1,26 +1,11 @@
...
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width">
-
-  <link rel="stylesheet" href="css/normalize.min.css">
-  <link rel="stylesheet" href="css/main.css">
-  <link rel="stylesheet" href="../../node_modules/video.js/dist/video-js.css">
-
-  <script src="js/vendor/modernizr-2.6.2.min.js"></script>
+  <link rel="stylesheet" href="switcher.css">
+  <link rel="stylesheet" href="/node_modules/video.js/dist/video-js.css">
```
Comment (forbesjo) [inline utils/switcher/index.html]: Missing `../..`

File: package.json [hunk 2]
```diff
@@ -40,9 +40,10 @@
     "version": "npm run build",
     "watch": "npm-run-all -p watch:*",
     "watch:docs": "nodemon --watch src/ --exec npm run docs",
-    "watch:js": "npm-run-all -p watch:js:babel watch:js:browserify",
+    "watch:js": "npm-run-all -p watch:js:babel watch:js:browserify watch:js:switcher",
     "watch:js:babel": "npm run build:js:babel -- --watch",
     "watch:js:browserify": "watchify . -v -g browserify-shim -o dist/videojs-contrib-hls.js",
+    "watch:js:switcher": "watchify utils/switcher/switcher.js -v -t babelify -g browserify-shim -o dist/switcher.js",
```
Comment (forbesjo) [inline package.json]: Shouldn't we add the browserify command to the build scripts as well?

File: utils/switcher/index.html (hunk 1 above)
Comment (forbesjo) [inline utils/switcher/index.html]: Ignore my comment, `npm start` will handle this

File: package.json (hunk 2 above)
Comment (forbesjo) [inline package.json]: Same as my above comment, `npm start` will handle this

**Comment by forbesjo** [comment]
Looks good to me

**Comment by BrandonOCasey** [comment]
once this is tested I will merge this PR

**Pull request merged**